test/types/TestMRSortModel.cpp

```diff
@@ -0,0 +1,78 @@
+#include "../../include/types/Criteria.h"
+#include "../../include/types/MRSortModel.h"
+#include "../../include/types/PerformanceTable.h"
+#include "gtest/gtest.h"
+#include <sstream>
+#include <utility>
+
+//        crit0  crit1  crit2
+// cat0
+//         0.8     1     0.4
+// cat1
+//         0.8    0.1    0.3
+// cat2
+//         0.6     0     0.2
+// base
+
+Criteria getTestCriteria() {
+  std::vector<Criterion> crit_vect;
+  crit_vect.push_back(Criterion("crit0", -1, 0.5));
+  crit_vect.push_back(Criterion("crit1", -1, 0.2));
+  crit_vect.push_back(Criterion("crit2", -1, 0.3));
+  return Criteria(crit_vect);
+}
+
+PerformanceTable getTestProfile() {
+  std::vector<Performance> perf_vect;
+  Criteria crit = getTestCriteria();
+  std::vector<float> given_perf0 = {0.8, 1, 0.4};
+  std::vector<float> given_perf1 = {0.8, 0.1, 0.3};
+  std::vector<float> given_perf2 = {0.6, 0, 0.2};
+  perf_vect.push_back(Performance(crit, given_perf0, "cat0"));
+  perf_vect.push_back(Performance(crit, given_perf1, "cat1"));
+  perf_vect.push_back(Performance(crit, given_perf2, "cat2"));
+  return PerformanceTable(perf_vect);
+}
```

Comment:
Do those functions belong in the test file ?

Reply:
Yes they just generate some data, it makes my tests a bit more readable and can be reused for further tests as well :)